pkg/kubelet/kubelet_node_status.go

```diff
@@ -515,6 +517,8 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 		}
 		node.Status.Allocatable[k] = value
 	}
+
+	node.Status.Allocatable[v1.ResourceNvidiaGPU] = *resource.NewQuantity(int64(kl.nvidiaGPUManager.AvailableGPUs()), resource.DecimalSI)
```

Comment:
And the Resource struct:

https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/scheduler/schedulercache/node_info.go#L66